fiftyone/core/stages.py

```
@@ -7664,6 +7947,9 @@ def _kwargs(self):
             ["create_index", self._create_index],
         ]
 
+    def _expressions(self):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_expressions`

(ANN202)